src/mixins/_button.scss

```diff
@@ -16,18 +15,26 @@
 
 @include theme.props {
   --theme-color-background-button-default: #{color.$brand-primary};
-  --theme-color-background-button-secondary: transparent;
+  --theme-color-background-button-primary: #{color.$base-purple};
+  --theme-color-background-button-secondary: var(--theme-color-background-base);
   --theme-color-border-button-default: #{color.$brand-primary-dark};
+  --theme-color-border-button-primary: #{color.$base-purple-dark};
   --theme-color-border-button-secondary: #{color.$brand-primary-light};
+  --theme-color-text-button-default: #{color.$text-light-emphasis};
+  --theme-color-text-button-primary: #{color.$text-light-emphasis};
   --theme-color-text-button-secondary: var(--theme-color-text-action);
   --theme-color-text-button-tertiary: var(--theme-color-text-action);
 }
 
 @include theme.props(dark) {
   --theme-color-background-button-default: #{color.$brand-primary-dark};
-  --theme-color-background-button-secondary: transparent;
+  --theme-color-background-button-primary: #{color.$base-white};
+  --theme-color-background-button-secondary: var(--theme-color-background-base);
   --theme-color-border-button-default: #{color.$brand-primary-darker};
-  --theme-color-border-button-secondary: #{color.$brand-primary-dark};
+  --theme-color-border-button-primary: #{color.$brand-primary-darker};
+  --theme-color-border-button-secondary: #{color.$brand-primary-darker};
+  --theme-color-text-button-default: #{color.$text-light-emphasis};
+  --theme-color-text-button-primary: #{color.$brand-primary};
   --theme-color-text-button-secondary: var(--theme-color-text-action);
   --theme-color-text-button-tertiary: var(--theme-color-text-action);
 }
```

Comment:
I think there's a future opportunity to simplify this a bit, but for now I didn't want to rock the boat and make this a much larger PR.